[OpenLdap] Adding AD instructions (#4991)
DataDog/integrations-core: openldap/README.md

#### Configure the OpenLDAP integration

##### Host

Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section.

###### Metric collection

> 1. Add this configuration block to your `openldap.yaml` file to start gathering your [metrics](#metrics). See the [sample openldap.yaml][2] for all available configuration options.

Reviewer: ```suggestion
1. Edit your `openldap.d/conf.yaml` in the `conf.d` folder at the root of your Agent's configuration directory. See the [sample openldap.d/conf.yaml][2] for all available configuration options.
```

    ```yaml

      init_config:

      instances:
          ## @param url - string - required
          ## Full URL of your ldap server. Use `ldaps` or `ldap` as the scheme to
          ## use TLS or not, or `ldapi` to connect to a UNIX socket.
          #
        - url: ldaps://localhost:636

          ## @param username - string - optional
          ## The DN of the user that can read the monitor database.
          #
          username: "<USER_DISTINGUISHED_NAME>"

          ## @param password - string - optional
          ## Password associated with `username`
          #
          password: "<PASSWORD>"
    ```

> 2. [Restart the Agent][3] to begin sending OpenLDAP metrics to Datadog.

Reviewer: ```suggestion
2. [Restart the Agent][3].
```

3. [Restart the Agent][3].

##### Containerized
###### Metric collection

For containerized environments, see the [Autodiscovery Integration Templates][4] for guidance on applying the parameters below.

| Parameter            | Value                                                                                         |
|----------------------|-----------------------------------------------------------------------------------------------|
| `<INTEGRATION_NAME>` | `openldap`                                                                                    |
| `<INIT_CONFIG>`      | blank or `{}`                                                                                 |

> | `<INSTANCE_CONFIG>`  | `"url":"ldaps://%%host%%:636","username":"<USER_DISTINGUISHED_NAME>","password":"<PASSWORD>"` |

Reviewer: ```suggestion
| `<INSTANCE_CONFIG>`  | `{"url":"ldaps://%%host%%:636","username":"<USER_DISTINGUISHED_NAME>","password":"<PASSWORD>"}` |
```
